module/app/upgrades/polaris/handler.go

```diff
@@ -0,0 +1,89 @@
+package polaris
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	ibctransferkeeper "github.com/cosmos/ibc-go/v3/modules/apps/transfer/keeper"
+	ibctransfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+)
+
+func GetPolarisUpgradeHandler(
+	mm *module.Manager, configurator *module.Configurator, transferKeeper *ibctransferkeeper.Keeper,
+) func(
+	ctx sdk.Context, plan upgradetypes.Plan, vmap module.VersionMap,
+) (module.VersionMap, error) {
+	if mm == nil || transferKeeper == nil {
+		panic("Nil argument to GetPolarisUpgradeHandler")
+	}
+	return func(ctx sdk.Context, plan upgradetypes.Plan, vmap module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info("Polaris upgrade: Enter handler")
+		// We previously upgraded via genesis, thus we don't want to run upgrades for all the modules
+		fromVM := make(map[string]uint64)
+		ctx.Logger().Info("Polaris upgrade: Creating version map")
+		for moduleName, module := range mm.Modules {
+			fromVM[moduleName] = module.ConsensusVersion()
+		}
+
```

Comment:
This is included so we don't lose track of it, Interchain Accounts can't be activated without running this commented upgrae logic. We will need to run it when we care to activate, and we must pay attention to the upgrade guide depending on the kind of functionality (controller, host, controller+host) we want to activate.